src/Native/Windows/clrcompression/CMakeLists.txt

```diff
@@ -40,6 +40,32 @@ elseif($ENV{__BuildArch} STREQUAL arm OR $ENV{__BuildArch} STREQUAL arm64)
     )
 endif()
 
+set (NATIVECOMPRESSION_SOURCES
```

Comment:
This will build for UWP, but we are not shipping it on UWP. Omit it for UWP?

Reply:
We can exclude these for "__AppContainer" builds but we don't have a notion of TargetGroup in the native build.

Reply:
Sounds good to me.